npm_and_yarn/helpers/package.json

```diff
@@ -12,7 +12,7 @@
     "@dependabot/yarn-lib": "^1.21.1",
     "@npmcli/arborist": "^2.0.3",
     "detect-indent": "^6.0.0",
-    "npm": "6.14.11",
+    "npm6": "npm:npm@6.14.11",
```

Comment:
@jurre just pushed this change to alias npm to `npm6`

Reply:
Nice!